docs/cromwell_features/CallCaching.md

```diff
@@ -81,6 +87,38 @@ Cromwell would search cache hits in all of the `gs://alice_bucket`, `gs://bob_bu
 
 If no `call_cache_hit_path_prefixes` are specified then all matching cache hits will be considered.
 
+***Call cache failure logging***
+
+By default when call caching fails to copy previous results the failure reason will be logged. Cromwell will continue to
+search for other call caching candidates, and when no candidates are left Cromwell will run the job on the backend.
+
+The logging of these informational messages can be adjusted via the configuration value:
+
+```hocon
+call-caching {
+  log-cache-hit-failures: true
+}
+```
+
+Set the above configuration value to `false` to disable the logging of call cache failure messages.
+
+***Call cache failure metadata***
+
+Similar to call cache failure logging, the informational messages may optionally be stored within the workflow metadata,
+where they will later be returned via the `/metadata` endpoint. By default the messages are not stored in the metadata,
+and are logged instead.
+
+```hocon
+call-caching {
+  add-cache-hit-failures-to-metadata: false
```

Comment:
Are we sure we need this? The ticket was pretty specific that there exists no customer requirement around recording to metadata, and I can't see ourselves really needing this as developers either.

Reply:
I like leaving this in and defaulting to `true` to avoid surprising anyone currently relying on it in other instances.

Reply:
Ditto on no more options for cromwell admins.